charts/victoria-logs-single/values.yaml

```diff
@@ -11,11 +11,13 @@ global:
   compatibility:
     openshift:
       adaptSecurityContext: "auto"
-  # -- Override chart name
```

Comment:
As I understand, you want to remove all `.global.nameOverride` usage, would you mind mentioning this in changelog?

Reply:
just removing it from docs to not confuse users

Reply:
I don't undersatnd, it's the values file you're changing, now both `.nameOverride` and `.global.nameOverride` options work, but only `.nameOverride` shows in the default values files we provide.